pdr_backend/pdr_dashboard/callbacks/callbacks_predictoors.py

```diff
@@ -80,6 +82,16 @@ def filter_table(
         if new_table_data:
             columns = get_feed_column_ids(new_table_data[0])
 
+        if sort_by:
+            # Extract sort criteria
+            sort_col = sort_by[0]["column_id"]
+            ascending = sort_by[0]["direction"] == "asc"
+
+            # Sort by raw "Price" even if the "Formatted Price" is displayed
+            new_table_data = sorted(
```

Comment:
does it take into account natural language? I.e. "2K" > "2M"?

Reply:
it uses the not formatted values to sort, so it should work fine